README.md

```diff
@@ -59,44 +100,82 @@ function NewComponent(props) {
 
 Sometimes you may have conflicts, perhaps with `React-Router`. In that case, you can alias your imports:
 
-```
-import * as DS from '@nypl/design-system-react-components';
-import { Link } from 'react-router'
+```jsx
+import * as DS from "@nypl/design-system-react-components";
+import { Link } from "react-router";
 
 function NewComponent(props) {
-  return <DS.Link><Link to="/license">public domain</Link></DS.Link>;
+  return (
+    <DS.Link>
+      <Link to="/license">public domain</Link>
+    </DS.Link>
+  );
 }
 ```
 
-Some things in the Design System, such as the universal focus styling and the box-model the DS uses, are namespaced to the `.nypl-ds` class. To include those in your app, add `.nypl-ds` to whichever wrapper tag makes sense in your application (e.g., `<div class="app">` or `<div class="container">`).
+Some CSS rules in the Design System, such as the universal focus styling and the box-model, are namespaced to the `.nypl-ds` class. To include those in your app, add `.nypl-ds` to whichever wrapper tag makes sense in your application (e.g., `<div class="app">` or `<div class="container">`).
+
+```jsx
+<div class="app nypl-ds">
+  <!-- your code here -->
+</div>
+```
 
-#### NYPL DS, NYPL Header, and NYPL Footer
+### NYPL DS, NYPL Header, and NYPL Footer
 
 Please note that the NYPL Header and Footer should be _outside_ of the `.nypl-ds` wrapper class.
 
-### CDN
+```jsx
+<body>
+  <NYPLHeader />
+
+  <div class="app nypl-ds">
+    <!-- your code here -->
+  </div>
+
+  <NYPLFooter />
+</body>
+```
+
+## CDN
 
-You can also use the Design System styles in your project with our CDN:
+You can also use the Design System styles in your project through the `unpkg` CDN:
 
-`<link href="https://unpkg.com/@nypl/design-system-react-components/dist/styles.css">`
+```jsx
+<link href="https://unpkg.com/@nypl/design-system-react-components/dist/styles.css">
+```
 
 If you need to reference a particular version you can do do by including the version number in the URL:
 
-`<link href="https://unpkg.com/@nypl/design-system-react-components@0.9.1/dist/styles.css">`
+```jsx
+<link href="https://unpkg.com/@nypl/design-system-react-components@0.9.1/dist/styles.css">
+```
 
-You can check out a working Codepen with unpkg [here](https://codepen.io/scoooooooby/pen/wvGzebO).
+You can check out a working Codepen with unpkg [here](https://codepen.io/edwinguzman/pen/ExmXGKx).
 
-## Design System and Product Requirements
+## Accessibility Product Requirements
 
 The NYPL Design System provides accessible stories, but real live data can necessitate additional accessibility requirements beyond what we're committed to in our generic, extensible components. To ensure your products' final result is accessible, please adhere to the accessibility requirements put together by NYPL's accessibility coordinator on [Metronome](http://themetronome.co/).
```

Comment:
Metronome is now password protected.  Maybe add something like, "For access to Metronome, please contact..."